__tests__/servertesting.test.js

```diff
@@ -41,12 +44,45 @@ describe("/get/api ", () => {
     return request(app)
       .get("/api")
       .expect(200)
-      .then((data) => {
-        data = JSON.parse(data.text);
-        console.log(data);
-        expect(data).toHaveProperty("GET /api");
-        expect(data).toHaveProperty("GET /api/topics");
-        expect(data).toHaveProperty("GET /api/articles");
+      .then(({ body }) => {
+        expect(body).toEqual(data);
+      });
+  });
+});
+describe("/get/api/articles/:article_id ", () => {
+  it("200: Should return an article object selected by a parametric endpoint, with apropriate keys", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then(({ body }) => {
+        body = body.articles;
+        expect(body.length > 0);
```

Comment:
jest expects us to use it's matcher functions like .toBe, not.toBe or .toBeGreaterThan - to set up this expectation correctly let's integrate an appropriate matcher